npm-shrinkwrap.json

```diff
@@ -1,70 +1,72 @@
 {
   "name": "perseo",
-  "version": "0.4.0-next",
+  "version": "0.4.1-next",
   "dependencies": {
     "async": {
       "version": "0.9.0",
-      "from": "async@^0.9.0",
+      "from": "https://registry.npmjs.org/async/-/async-0.9.0.tgz",
       "resolved": "https://registry.npmjs.org/async/-/async-0.9.0.tgz"
     },
     "body-parser": {
       "version": "1.10.1",
-      "from": "body-parser@^1.5.2",
+      "from": "https://registry.npmjs.org/body-parser/-/body-parser-1.10.1.tgz",
```

Comment:
After a second look at this, `version` and `resolved` haven't changed, so it seems to be ok. NTC.